src/gui/optionsdlg.cpp

```diff
@@ -95,7 +99,13 @@ OptionsDialog::OptionsDialog(QWidget *parent)
 #endif
     m_ui->tabSelection->item(TAB_ADVANCED)->setIcon(GuiIconProvider::instance()->getIcon("preferences-other"));
     for (int i = 0; i < m_ui->tabSelection->count(); ++i) {
-        m_ui->tabSelection->item(i)->setSizeHint(QSize(std::numeric_limits<int>::max(), 64));  // uniform size for all icons
+        // uniform size for all icons
+#ifdef Q_OS_MAC
+        // Smaller size for mac due to rendering specifics to avoid vertical scroll
```

Comment:
To be honest, I do not think that increasing the dialogue height is a good idea. It already is 615 pixels tall here, plus 75 pixels for the Dock and 25 for the menubar. This makes it pretty big already for smaller screens (e.g. when remoting from iPad, which has 1024x768 res). I reduced the icon size instead.